Dialog for generating scoped tokens

fkorotkov opened the pull request:
Fixes https://github.com/cirruslabs/cirrus-ci-docs/issues/1080

edigaryev commented:
LGTM overall.

The code attempts to handle a GraphQL error condition when generating a token, yet still throws an exception when entering an incorrect list of comma-separated repositories, for example, `non-existent` or `cirrus-playground,`:

<img width="1020" alt="Screenshot 2022-11-01 at 11 26 57" src="https://user-images.githubusercontent.com/85709/199181139-5aa2c2a2-610e-4b72-abbd-3bb95d88c7bb.png">

The other minor issue I've found is when clicking the "Expiration in days (Optional)" input, the label gets collided with the element above it:

<img width="527" alt="Screenshot 2022-11-01 at 11 30 16" src="https://user-images.githubusercontent.com/85709/199181697-56a042a7-a117-4089-a033-4a117fc58a65.png">

fkorotkov commented:
It seems relay changed error handling at some point. Fixed everywhere in [e84b93c](https://github.com/cirruslabs/cirrus-ci-web/pull/458/commits/e84b93c70cdc93bb62cc220db978fb3f4f07e6bb). Also fixed margins in [854cbad](https://github.com/cirruslabs/cirrus-ci-web/pull/458/commits/854cbadee5f43a7003760dea3598c523aa5d7638). PTAL
